### DISCUSSION TITLE
Postscript - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#338] [Announcements]
# Postscript - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/%E5%90%8E%E8%AE%B0/

**Comment by XXCDMXX** [comment]
🥺🥺🥺

**Comment by XindaLi304** [comment]
goat

**Comment by taotaoboat** [comment]
很棒哎。cs是一个交流促进的领域，感觉自学缺少一点讨论的机会和对大方向的把控。不知道题主到了master或者phd的时候会怎么想。有时候除了技能和课时还需要点别的。

**Comment by xuxiaominger** [comment]
Ever since high school, I’ve been a liberal arts student. I majored in law at college, and my job is also in the humanities. But when I stumbled across this post on YouTube, it felt like I’d discovered a hidden gem.

Computer science gave me a way to pull myself out of the endless grind of tedious legal work and sharp human conflicts — into a quiet space where I could think deeply and keep learning. The author’s content is dense yet efficient, clearly organized with care and real-world experience, and above all, honest.

Even though I come from a liberal arts background and have zero foundation in computer science, I’ve learned a great deal from the author’s sincere and detailed explanations. I, too, have plenty of frustrations with my own university, but seeing someone so dedicated to refining and sharing knowledge reminds me that our generation still has people with genuine conscience and heart.
